src/goto-harness/function_call_harness_generator.cpp

```diff
@@ -40,14 +45,17 @@ struct function_call_harness_generatort::implt
   recursive_initialization_configt recursive_initialization_config;
   std::unique_ptr<recursive_initializationt> recursive_initialization;
 
+  std::set<irep_idt> function_parameters_to_treat_as_arrays;
+  std::set<irep_idt> function_arguments_to_treat_as_arrays;
```

Comment:
❓ Why `std::set` instead of `std::unordered_set`. Given what you're doing the former would be much more efficient.

Reply:
In practical terms I don't expect these will ever contain more than _maybe_ a couple dozen or so members, so don't know about _much_ more efficient. But it's not like changing to `unordered_` is going to break anything, so I don't mind it either.